scripts/ghidra/ScriptWrapper.java

```diff
@@ -232,7 +245,8 @@ Signature getBromaSignature(Broma.Function fun) throws Exception {
         }
         // Parse return type, or null if this is a destructor
         ReturnParameterImpl bromaRetType = null;
-        if (fun.returnType.isPresent() && !fun.returnType.get().name.value.contains("TodoReturn")) {
+        // If we want to set TodoReturn in ghidra, set bromaRetType even if the function returnType is TodoReturn
+        if (fun.returnType.isPresent() && (this.wrapped.args.setTodoReturnInGhidra || !fun.returnType.get().name.value.contains("TodoReturn"))) {

```

Comment:
Ok after looking into this, this should definitely just be default behaviour and not an option